### ISSUE TITLE
[BUG] terrain console is unable to execute lib functions in example template

### ISSUE THREAD

**Issue by dominictwlee** [terra-money/terrain#101]
### Issue description
When trying to execute the functions provided in the lib/index.ts file via terrain console, I receive an error regarding undefined contractAddresses from ContractRefs.

### Steps to reproduce the issue

1. Setup new dapp template (terrain new example_dapp)
2. Start up local Terra
3. terrain deploy example_dapp
4. terrain console > lib.increment()
5. See error.

### Tools and operating system versions
8.11.0
v16.16.0
docker-compose version 1.27.2, build 18f557f9
Docker version 20.10.15, build fd82621
@terra-money/terrain/0.4.1 linux-x64 node-v16.16.0

### Additional details / screenshot
![image](https://user-images.githubusercontent.com/34794189/178772034-d3e5a4a3-9b69-42d1-8032-c63fc20e6436.png)

**Comment by dominictwlee** [comment]
Did a bit of debugging, it seems "terrain new" creates the first contract with the same dapp name instead of counter, but when executing from terrain console, it's passed "counter" as the contract string:

![image](https://user-images.githubusercontent.com/34794189/178778169-81e1119c-f5b4-47eb-8f12-23d5f6d5e116.png)

**Comment by emidev98** [comment]
There is a release candidate that fixes this issue. I will keep you updated when the new version is released

**Comment by octalmage** [comment]
@dominictwlee just released v0.5.6 which includes a lot of updates, including a fix for this. 

First I'd scaffold a new project since the core templates have changed a bit. 

The console and frontend work a little differently now, after deploying your contract you should run: 

```
terrain contract:generateClient <contract-name> --build-schema
```

This will generate a typescript client to be used in the console. 

Let me know if this works for you!
